core/templates/services/audio-player.service.ts

```
@@ -97,16 +103,18 @@ export class AudioPlayerService {
 
     this.ngZone.runOutsideAngular(() => {
       if (this._currentTrack !== null) {
-        this._currentTrack.seek(this._lastPauseOrSeekPos);
+        // 'lastPauseOrSeekPos' will not be null since currentTrack exists.
+        this._currentTrack.seek(<number> this._lastPauseOrSeekPos);
       }
       interval(500).pipe(takeUntil(
         this._stopIntervalSubject)).subscribe(() => {
         this.ngZone.run(() => {
           this._updateViewEventEmitter.emit();
         });
       });
-
-      this._currentTrack.play();
+      if (this._currentTrack !== null) {
+        this._currentTrack.play();
```

Should we throw an error here is `_currentTrack` is null ?

---

yes, done!